docs/workflows/transitions/postprocess/tests/test_utils.py

```diff
@@ -71,3 +93,11 @@ def test_prediction_conversion(v1_preds):
     assert v2_preds['invoice_id'][0]['value'] == '54321'
     assert set(v2_preds['line_items'][0]['line_amount'][0].keys()) == expected_keys
     assert v2_preds['line_items'][0]['line_amount'][0]['value'] == '350.00'
+
+
+def test_to_validated_format(v2_preds):
+    validated = to_validated_format(v2_preds)
+
+    assert validated['total_amount']['value'] == v2_preds['total_amount'][0]['value']
+    assert validated['line_items'][0]['line_amount']['value'] == v2_preds['line_items'][0]['line_amount'][0]['value']
+    assert validated['line_items'][0]['description']['value'] == v2_preds['line_items'][0]['description'][0]['value']
```

Comment:
I think it would be wise to test the second line as well in case we make some unfortunate changes in the future
```suggestion
    assert validated['line_items'][0]['line_amount']['value'] == v2_preds['line_items'][0]['line_amount'][0]['value']
    assert validated['line_items'][0]['description']['value'] == v2_preds['line_items'][0]['description'][0]['value']
    assert validated['line_items'][1]['line_amount']['value'] == v2_preds['line_items'][1]['line_amount'][0]['value']
    assert validated['line_items'][1]['description']['value'] == v2_preds['line_items'][1]['description'][0]['value']
```